releases/2020.05.yaml

```diff
@@ -0,0 +1,125 @@
+repositories:
+  qpOASES:
+    type: git
+    url: https://github.com/robotology-dependencies/qpOASES.git
+    version: v3.2.0.1
+  osqp:
+    type: git
+    url: https://github.com/oxfordcontrol/osqp.git
+    version: v0.6.0
+  ycm:
+    type: git
+    url: https://github.com/robotology/ycm.git
+    version: v0.11.0
+  YARP:
+    type: git
+    url: https://github.com/robotology/yarp.git
+    version: v3.3.2
+  ICUB:
+    type: git
+    url: https://github.com/robotology/icub-main.git
+    version: v1.15.0
+  ICUBcontrib:
+    type: git
+    url: https://github.com/robotology/icub-contrib-common.git
+    version: v1.15.0
+  robots-configuration:
+    type: git
+    url: https://github.com/robotology/robots-configuration.git
+    version: v1.15.0
+  GazeboYARPPlugins:
+    type: git
+    url: https://github.com/robotology/gazebo-yarp-plugins.git
+    version: v3.3.0
+  icub-gazebo:
+    type: git
+    url: https://github.com/robotology/gazebo-yarp-plugins.git
+    version: v1.15.0
```

Comment:
wrong url

Reply:
Thanks, fixed in https://github.com/robotology/robotology-superbuild/pull/386/commits/a7fc0dda4ae275468cc18f36250d25355d0d1f27 .